api/apps/v1alpha1/nimcache_types.go

```diff
@@ -39,8 +39,8 @@ type NIMCacheSpec struct {
 	Resources Resources `json:"resources,omitempty"`
 	// Tolerations for running the job to cache the NIM model
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
-	// NodeSelectors are the node selector labels to schedule the caching job.
-	NodeSelectors map[string]string `json:"gpuSelectors,omitempty"`
+	// NodeSelector is the node selector labels to schedule the caching job.
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
hmm, this will be a breaking change. But `gpuSelectors` is confusing, we need to call this out in the documentation.

Reply:
Yes. Cant think any better way to do it :)